otx/algorithms/classification/adapters/mmcls/data/datasets.py

```diff
@@ -387,9 +387,9 @@ def evaluate(
         for i in range(self.hierarchical_info["num_multiclass_heads"]):
             multiclass_logit = results[
                 :,
-                self.hierarchical_info["head_idx_to_logits_range"][i][0] : self.hierarchical_info[
+                self.hierarchical_info["head_idx_to_logits_range"][str(i)][0] : self.hierarchical_info[
```

Comment:
Why this change is required? The data type is supposed to be integer here. Is this change related to the other ones in this PR? Previously, we had no any issues with this code, as far, as I know.